corehq/apps/export/views.py

```diff
@@ -2016,10 +2008,6 @@ class DownloadNewSmsExportView(GenericDownloadNewExportMixin, BaseDownloadExport
     export_id = None
     sms_export = True
 
-    @staticmethod
-    def get_export_schema(domain, include_metadata):
-        return SMSExportDataSchema.get_latest_export_schema(domain, include_metadata)
```

Comment:
I don't know what the deal with this method was - none of the callers match this method signature.

Reply:
Odd, even the [original implementation](https://github.com/dimagi/commcare-hq/pull/14823/files#diff-1086f45b5d86ddf7a9cd7b848caf29a6R2298) doesn't have any callers that match.